src/app/components/home/thumbnail/preview.component.ts

```diff
@@ -51,8 +54,30 @@ export class PreviewComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.firstFilePath = 'file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/thumbnails/' + this.video.hash + '.jpg';
-    this.fullFilePath =  'file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/filmstrips/' + this.video.hash + '.jpg';
+    // multiple hashes?
+    if (this.video.hash.indexOf(':') !== -1) {
+      const hashes = this.video.hash.split(':');
+      this.shuffle(hashes).slice(0, 4).forEach((hash) => {
+        this.folderThumbPaths.push('file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/thumbnails/' + hash + '.jpg');
+      });
+      this.numberOfThumbs = this.folderThumbPaths.length;
+    } else {
+      this.firstFilePath = 'file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/thumbnails/' + this.video.hash + '.jpg';
+      this.fullFilePath =  'file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/filmstrips/' + this.video.hash + '.jpg';
+      this.folderThumbPaths.push(this.firstFilePath);
+      this.numberOfThumbs = 1;
+    }
+  }
+
+  shuffle(a) {
```

Comment:
This method appears exactly the same in `clip.component` which is fine for now, but a few minor recommendations:
- include `JSDoc` to briefly describe what the method does
- add TypeScript to the input (so it's clear what kind of objects are allowed to be used as input)
- place the code for this method into some helper `service` and refer to the method from both places

All three can be accomplished later after the merge 👌